### PR TITLE
installer update readme

### PR DESCRIPTION
**Pull request by florianGla**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+### Building from Source (Linux)
```
Comment (jochil) [inline CONTRIBUTING.md:3]:
```suggestion
## Building from Source (Linux)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+**Prerequisites**
+* [git](https://git-scm.com/)
+* [make](https://www.gnu.org/software/make/)
+* [CMake >= 3.16](https://cmake.org/)
```
Comment (jochil) [inline CONTRIBUTING.md:8]: For development you need CMake v3.21... we discussed it here: https://codeintelligence.slack.com/archives/C03GJGGP69H/p1658404171321049

File: README.md [hunk 3]
```diff
@@ -19,46 +19,31 @@ look at our [Glossary](docs/Glossary.md).
...
-```
-If everything went fine, you will find the newly created directory
-`~/cifuzz`. Do not forget to add `~/cifuzz/bin` to your `$PATH`.
-
-To verify the installation we recommend you to start a fuzzing run
-in one of our example projects:
-``` bash
-cd examples/cmake
-cifuzz run my_fuzz_test
+chmod +x cifuzz_installer && ./cifuzz_installer
```
Comment (jochil) [inline README.md:40]: we can also add a `wget` or `curl` command to download the latest installer... you can find the permanent links in slack too

File: README.md [hunk 4]
```diff
@@ -19,46 +19,31 @@ look at our [Glossary](docs/Glossary.md).
...
 ```bash
-sudo apt install git make cmake clang llvm golang-go libcap-dev
+sudo apt install cmake clang llvm
 ```
-
 **Arch**
 <!-- when changing this, please make sure it is in sync with the E2E pipeline -->
 ```bash
-sudo pacman -S git make cmake clang llvm go libcap
+sudo pacman -S git cmake clang llvm
```
Comment (jochil) [inline README.md:34]: `git` is not needed

File: README.md (hunk 3 above)
Comment (florianGla) [reply README.md:40]: Good idea. Shall we only offer a optional wget command for linux then?

Comment (jochil) [reply README.md:40]: I think `curl` is more common... I remember systems where I had to add `wget` by hand. `curl` should work also on mac. I think we can do something like this
```
curl -L -o cifuzz_installer https://github.com/CodeIntelligenceTesting/cifuzz/releases/latest/download/cifuzz_installer_linux # or _darwin for MacOS
chmod +x cifuzz_installer && ./cifuzz_installer
```

File: README.md [hunk 5]
```diff
@@ -19,46 +19,31 @@ look at our [Glossary](docs/Glossary.md).
...
-
 
 **Ubuntu / Debian**
 <!-- when changing this, please make sure it is in sync with the E2E pipeline -->
 ```bash
-sudo apt install git make cmake clang llvm golang-go libcap-dev
+sudo apt install cmake clang llvm
 ```
-
 **Arch**
```
Comment (jochil) [inline README.md:31]: maybe now we can add at least MacOS with a Homebrew command
```
brew install llvm cmake
```

Comment (florianGla) [reply README.md:31]: Maybe we could offer a [choco](https://github.com/chocolatey/choco) command. LLVM and CMake are available, but I never tried it.

Comment (jochil) [reply README.md:31]: Would be ok for me

**Pull request merged**